sdk/applicationinsights/microsoft-azure-applicationinsights-query/samples/pom.xml

```diff
@@ -9,10 +9,16 @@
     <modelVersion>4.0.0</modelVersion>
     <groupId>com.microsoft.azure</groupId>
     <artifactId>azure-applicationinsights-query-sample</artifactId>
-    <version>1.0.0-Preview-1</version>
-    <packaging>jar</packaging>
+    <version>1.0.0-Preview-1</version> <!-- {x-version-update;com.microsoft.azure:azure-applicationinsights-query-sample;current} -->
+  <packaging>jar</packaging>
     <name>Microsoft Azure SDK for Application Insights Query Sample</name>
     <description>This package contains Microsoft Application Insights Query SDK Sample.</description>
+    <parent>
+        <groupId>com.azure</groupId>
+        <artifactId>azure-data-sdk-parent</artifactId>
+        <version>1.3.0</version> <!-- {x-version-update;com.azure:azure-data-sdk-parent;current} -->
+        <relativePath>../../parents/azure-data-sdk-parent</relativePath>
```

Comment:
Do we need this for `samples/pom.xml`?

Reply:
When I started this work I found that if I didn't include the parent build for these samples would fail. I can go back and check if they can be removed.

Reply:
Other sample packages (especially Track 2) inherit from the Parent POM. I don't think it hurts to align these as well.